dockstore-integration-testing/src/test/java/io/dockstore/client/cli/OrganizationIT.java

```diff
@@ -331,6 +334,31 @@ public void testCreateNewOrganization() {
         assertEquals("potato", organization.getDescription());
         String description = organizationsApiUser2.getOrganizationDescription(organization.getId());
         assertEquals("potato", description);
+
+        testStarredOrganizationEvents(organizationsApiUser2, organization);
+    }
+
+    private void testStarredOrganizationEvents(OrganizationsApi organizationsApiUser2, Organization organization) {
+        organizationsApiUser2.starOrganization(organization.getId(), SwaggerUtility.createStarRequest(true));
+        final io.dockstore.openapi.client.ApiClient openAPIWebClientUser2 = getOpenAPIWebClient(USER_2_USERNAME, testingPostgres);
+        EventsApi eventsApi = new EventsApi(openAPIWebClientUser2);
+        List<io.dockstore.openapi.client.model.Event> events = eventsApi
+                .getEvents(EventSearchType.STARRED_ORGANIZATION.toString(), null, null);
+        Assert.assertEquals("Should have 1 create, 1 approve, 4 modify events", 6, events.size());
```

Comment:
A bit misleading, the test only checks that there are 6 events in general. 
Same idea with two lines down

Reply:
It was only supposed to test for 6 events.  The comment was only to provide additional details for debugging purposes.

Reply:
Understood, what I meant was that the test would be better if it actually checked for 1 create, 1 approve, 4 modify events. 
Right now, it looks like the test could pass if it got 6 create events.

Reply:
Yes, that's what it's intended to do.  I'm testing whether the pagination works for this endpoint not whether the event type was correctly created.  We have other tests for that.